client/my-sites/activity/controller.jsx

```diff
@@ -71,7 +67,7 @@ export function activity( context, next ) {
 	}
 
 	recordTrack( 'calypso_activitylog_view', queryFilterToStats( queryFilter ) );
-	context.primary = <ActivityLog siteId={ siteId } context={ context } startDate={ startDate } />;
+	context.primary = <ActivityLog siteId={ siteId } context={ context } />;
```

Comment:
The `startDate` prop really isn't used anywhere inside `<ActivityLog>`?

Reply:
No, apparently not. A quick find yields no results, and a deeper inspection indicates that the props object isn't being forwarded to any other component.

Reply:
The `startDate` feature was indeed removed in #23898 and also in #30931. Thanks @seear for hunting it down! :+1: